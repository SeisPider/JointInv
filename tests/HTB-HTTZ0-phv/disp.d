SURF96 R U T  0          95       3.569      0.1899 
SURF96 R U T  0          90       3.645      0.1876 
SURF96 R U T  0          85       3.725      0.1851 
SURF96 R U T  0          80       3.773      0.1787 
SURF96 R U T  0          75       3.759      0.1663 
SURF96 R U T  0          70       3.719      0.1519 
SURF96 R U T  0          65       3.681      0.1382 
SURF96 R U T  0          60       3.652      0.1256 
SURF96 R U T  0          55       3.629      0.1136 
SURF96 R U T  0          50         3.6      0.1017 
SURF96 R U T  0          48        3.58     0.09654 
SURF96 R U T  0          46       3.552     0.09109 
SURF96 R U T  0          44       3.514     0.08526 
SURF96 R U T  0          42       3.468     0.07924 
SURF96 R U T  0          40       3.415     0.07321 
SURF96 R U T  0          38       3.362      0.0674 
SURF96 R U T  0          36       3.306     0.06173 
SURF96 R U T  0          34       3.246      0.0562 
SURF96 R U T  0          32       3.186     0.05097 
SURF96 R U T  0          30       3.133     0.04621 
SURF96 R U T  0          29       3.112     0.04406 
SURF96 R U T  0          28       3.097     0.04215 
SURF96 R U T  0          27       3.089     0.04043 
SURF96 R U T  0          26       3.086     0.03885 
SURF96 R U T  0          25       3.088      0.0374 
