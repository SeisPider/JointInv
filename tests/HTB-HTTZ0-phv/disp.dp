SURF96 R U T  0       93.29       3.569      0.1899 
SURF96 R U T  0       89.38       3.645      0.1876 
SURF96 R U T  0       84.54       3.725      0.1851 
SURF96 R U T  0       78.94       3.773      0.1787 
SURF96 R U T  0       73.61       3.759      0.1663 
SURF96 R U T  0       68.76       3.719      0.1519 
SURF96 R U T  0        64.1       3.681      0.1382 
SURF96 R U T  0       59.35       3.652      0.1256 
SURF96 R U T  0       54.46       3.629      0.1136 
SURF96 R U T  0        49.6         3.6      0.1017 
SURF96 R U T  0       47.67        3.58     0.09654 
SURF96 R U T  0       45.72       3.552     0.09109 
SURF96 R U T  0        43.7       3.514     0.08526 
SURF96 R U T  0       41.66       3.468     0.07924 
SURF96 R U T  0       39.62       3.415     0.07321 
SURF96 R U T  0       37.62       3.362      0.0674 
SURF96 R U T  0       35.63       3.306     0.06173 
SURF96 R U T  0       33.59       3.246      0.0562 
SURF96 R U T  0       31.68       3.186     0.05097 
SURF96 R U T  0       29.94       3.133     0.04621 
SURF96 R U T  0        29.1       3.112     0.04406 
SURF96 R U T  0       28.31       3.097     0.04215 
SURF96 R U T  0        27.6       3.089     0.04043 
SURF96 R U T  0       26.98       3.086     0.03885 
SURF96 R U T  0       26.46       3.088      0.0374 
