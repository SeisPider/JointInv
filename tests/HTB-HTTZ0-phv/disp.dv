MFT96 R U T  0 95.256    3.6524    1.0000    6373.2     0.5092E-06   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 93.091    3.6602    1.0000    6373.2     0.5684E-06   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 91.022    3.6674    1.0000    6373.2     0.6309E-06   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 89.043    3.6740    1.0000    6373.2     0.6972E-06   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 87.149    3.6801    1.0000    6373.2     0.7678E-06   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 85.333    3.6857    1.0000    6373.2     0.8436E-06   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 83.592    3.6907    1.0000    6373.2     0.9251E-06   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 81.920    3.6952    1.0000    6373.2     0.1013E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 80.314    3.6992    1.0000    6373.2     0.1108E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 78.769    3.7026    1.0000    6373.2     0.1211E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 77.283    3.7055    1.0000    6373.2     0.1321E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 75.852    3.7079    1.0000    6373.2     0.1439E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 74.473    3.7098    1.0000    6373.2     0.1565E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 73.143    3.7113    1.0000    6373.2     0.1698E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 71.860    3.7122    1.0000    6373.2     0.1838E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 70.621    3.7126    1.0000    6373.2     0.1984E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 69.424    3.7126    1.0000    6373.2     0.2136E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 68.267    3.7121    1.0000    6373.2     0.2293E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 67.148    3.7112    1.0000    6373.2     0.2454E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 66.065    3.7098    1.0000    6373.2     0.2618E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 65.016    3.7080    1.0000    6373.2     0.2786E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 64.000    3.7057    1.0000    6373.2     0.2957E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 63.015    3.7031    1.0000    6373.2     0.3131E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 62.061    3.7001    1.0000    6373.2     0.3307E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 61.134    3.6966    1.0000    6373.2     0.3487E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 60.235    3.6928    1.0000    6373.2     0.3670E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 59.362    3.6887    1.0000    6373.2     0.3858E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 58.514    3.6841    1.0000    6373.2     0.4049E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 57.690    3.6793    1.0000    6373.2     0.4245E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 56.889    3.6741    1.0000    6373.2     0.4446E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 56.110    3.6686    1.0000    6373.2     0.4651E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 55.351    3.6628    1.0000    6373.2     0.4861E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 54.613    3.6567    1.0000    6373.2     0.5074E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 53.895    3.6503    1.0000    6373.2     0.5290E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 53.195    3.6436    1.0000    6373.2     0.5509E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 52.513    3.6367    1.0000    6373.2     0.5729E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 51.848    3.6295    1.0000    6373.2     0.5949E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 51.200    3.6221    1.0000    6373.2     0.6169E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 50.568    3.6145    1.0000    6373.2     0.6388E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 49.951    3.6067    1.0000    6373.2     0.6606E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 49.349    3.5986    1.0000    6373.2     0.6822E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 48.762    3.5904    1.0000    6373.2     0.7037E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 48.188    3.5820    1.0000    6373.2     0.7252E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 47.628    3.5735    1.0000    6373.2     0.7466E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 47.080    3.5648    1.0000    6373.2     0.7681E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 46.545    3.5559    1.0000    6373.2     0.7898E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 46.022    3.5469    1.0000    6373.2     0.8119E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 45.511    3.5378    1.0000    6373.2     0.8345E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 45.011    3.5286    1.0000    6373.2     0.8578E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 44.522    3.5193    1.0000    6373.2     0.8818E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 44.043    3.5099    1.0000    6373.2     0.9069E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 43.574    3.5004    1.0000    6373.2     0.9329E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 43.116    3.4908    1.0000    6373.2     0.9601E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 42.667    3.4812    1.0000    6373.2     0.9885E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 42.227    3.4715    1.0000    6373.2     0.1018E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 41.796    3.4618    1.0000    6373.2     0.1049E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 41.374    3.4521    1.0000    6373.2     0.1081E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 40.960    3.4423    1.0000    6373.2     0.1114E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 40.554    3.4326    1.0000    6373.2     0.1148E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 40.157    3.4228    1.0000    6373.2     0.1183E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 39.767    3.4130    1.0000    6373.2     0.1219E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 39.385    3.4032    1.0000    6373.2     0.1256E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 39.010    3.3935    1.0000    6373.2     0.1293E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 38.642    3.3838    1.0000    6373.2     0.1331E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 38.280    3.3741    1.0000    6373.2     0.1369E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 37.926    3.3644    1.0000    6373.2     0.1408E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 37.578    3.3549    1.0000    6373.2     0.1448E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 37.236    3.3453    1.0000    6373.2     0.1488E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 36.901    3.3358    1.0000    6373.2     0.1530E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 36.571    3.3264    1.0000    6373.2     0.1573E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 36.248    3.3171    1.0000    6373.2     0.1618E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 35.930    3.3079    1.0000    6373.2     0.1664E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 35.617    3.2987    1.0000    6373.2     0.1713E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 35.310    3.2897    1.0000    6373.2     0.1763E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 35.009    3.2807    1.0000    6373.2     0.1817E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 34.712    3.2719    1.0000    6373.2     0.1873E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 34.420    3.2631    1.0000    6373.2     0.1932E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 34.133    3.2545    1.0000    6373.2     0.1994E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 33.851    3.2460    1.0000    6373.2     0.2058E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 33.574    3.2377    1.0000    6373.2     0.2124E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 33.301    3.2295    1.0000    6373.2     0.2193E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 33.032    3.2214    1.0000    6373.2     0.2263E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 32.768    3.2134    1.0000    6373.2     0.2334E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 32.508    3.2057    1.0000    6373.2     0.2406E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 32.252    3.1980    1.0000    6373.2     0.2477E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 32.000    3.1906    1.0000    6373.2     0.2546E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 31.752    3.1833    1.0000    6373.2     0.2614E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 31.508    3.1762    1.0000    6373.2     0.2679E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 31.267    3.1692    1.0000    6373.2     0.2740E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 31.030    3.1624    1.0000    6373.2     0.2796E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 30.797    3.1558    1.0000    6373.2     0.2848E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 30.567    3.1494    1.0000    6373.2     0.2895E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 30.341    3.1432    1.0000    6373.2     0.2935E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 30.118    3.1372    1.0000    6373.2     0.2970E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 29.898    3.1314    1.0000    6373.2     0.2998E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 29.681    3.1258    1.0000    6373.2     0.3020E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 29.468    3.1205    1.0000    6373.2     0.3035E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 29.257    3.1153    1.0000    6373.2     0.3045E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 29.050    3.1103    1.0000    6373.2     0.3048E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 28.845    3.1056    1.0000    6373.2     0.3045E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 28.643    3.1011    1.0000    6373.2     0.3036E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 28.444    3.0969    1.0000    6373.2     0.3021E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 28.248    3.0929    1.0000    6373.2     0.3000E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 28.055    3.0891    1.0000    6373.2     0.2973E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 27.864    3.0855    1.0000    6373.2     0.2939E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 27.676    3.0823    1.0000    6373.2     0.2899E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 27.490    3.0793    1.0000    6373.2     0.2851E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 27.307    3.0765    1.0000    6373.2     0.2797E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 27.126    3.0740    1.0000    6373.2     0.2734E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 26.947    3.0718    1.0000    6373.2     0.2663E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 26.771    3.0699    1.0000    6373.2     0.2584E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 26.597    3.0682    1.0000    6373.2     0.2496E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 26.426    3.0668    1.0000    6373.2     0.2399E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 26.256    3.0658    1.0000    6373.2     0.2294E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 26.089    3.0650    1.0000    6373.2     0.2181E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 25.924    3.0645    1.0000    6373.2     0.2061E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 25.761    3.0643    1.0000    6373.2     0.1934E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 25.600    3.0645    1.0000    6373.2     0.1801E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 25.441    3.0650    1.0000    6373.2     0.1664E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 25.284    3.0658    1.0000    6373.2     0.1524E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 25.129    3.0670    1.0000    6373.2     0.1382E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 95.000    3.6533    1.0000    6373.2     0.5162E-06   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 90.000    3.6708    1.0000    6373.2     0.6651E-06   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 85.000    3.6867    1.0000    6373.2     0.8592E-06   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 80.000    3.6999    1.0000    6373.2     0.1129E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 75.000    3.7091    1.0000    6373.2     0.1517E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 70.000    3.7126    1.0000    6373.2     0.2063E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 65.000    3.7079    1.0000    6373.2     0.2789E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 60.000    3.6917    1.0000    6373.2     0.3721E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 55.000    3.6599    1.0000    6373.2     0.4962E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 50.000    3.6073    1.0000    6373.2     0.6589E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 48.000    3.5791    1.0000    6373.2     0.7323E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 46.000    3.5465    1.0000    6373.2     0.8129E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 44.000    3.5090    1.0000    6373.2     0.9092E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 42.000    3.4664    1.0000    6373.2     0.1034E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 40.000    3.4189    1.0000    6373.2     0.1198E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 38.000    3.3665    1.0000    6373.2     0.1400E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 36.000    3.3099    1.0000    6373.2     0.1654E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 34.000    3.2505    1.0000    6373.2     0.2024E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 32.000    3.1906    1.0000    6373.2     0.2546E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 30.000    3.1341    1.0000    6373.2     0.2985E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 29.000    3.1092    1.0000    6373.2     0.3047E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 28.000    3.0881    1.0000    6373.2     0.2963E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 27.000    3.0724    1.0000    6373.2     0.2684E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T  0 26.000    3.0647    1.0000    6373.2     0.2116E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
