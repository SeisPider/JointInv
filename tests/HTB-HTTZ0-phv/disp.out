SURF96 R U T  0 95.000    3.6533    1.0000    
SURF96 R C T  0 95.000    5.9930    1.0000    
SURF96 R U T  0 90.000    3.6708    1.0000    
SURF96 R C T  0 90.000    5.7989    1.0000    
SURF96 R U T  0 85.000    3.6867    1.0000    
SURF96 R C T  0 85.000    5.6189    1.0000    
SURF96 R U T  0 80.000    3.6999    1.0000    
SURF96 R C T  0 80.000    5.4518    1.0000    
SURF96 R U T  0 75.000    3.7091    1.0000    
SURF96 R C T  0 75.000    5.2957    1.0000    
SURF96 R U T  0 70.000    3.7126    1.0000    
SURF96 R C T  0 70.000    5.1492    1.0000    
SURF96 R U T  0 65.000    3.7079    1.0000    
SURF96 R C T  0 65.000    5.0104    1.0000    
SURF96 R U T  0 60.000    3.6917    1.0000    
SURF96 R C T  0 60.000    4.8777    1.0000    
SURF96 R U T  0 55.000    3.6599    1.0000    
SURF96 R C T  0 55.000    4.7484    1.0000    
SURF96 R U T  0 50.000    3.6073    1.0000    
SURF96 R C T  0 50.000    4.6197    1.0000    
SURF96 R U T  0 48.000    3.5791    1.0000    
SURF96 R C T  0 48.000    4.5675    1.0000    
SURF96 R U T  0 46.000    3.5465    1.0000    
SURF96 R C T  0 46.000    4.5145    1.0000    
SURF96 R U T  0 44.000    3.5090    1.0000    
SURF96 R C T  0 44.000    4.4603    1.0000    
SURF96 R U T  0 42.000    3.4664    1.0000    
SURF96 R C T  0 42.000    4.4044    1.0000    
SURF96 R U T  0 40.000    3.4189    1.0000    
SURF96 R C T  0 40.000    4.3466    1.0000    
SURF96 R U T  0 38.000    3.3665    1.0000    
SURF96 R C T  0 38.000    4.2864    1.0000    
SURF96 R U T  0 36.000    3.3099    1.0000    
SURF96 R C T  0 36.000    4.2232    1.0000    
SURF96 R U T  0 34.000    3.2505    1.0000    
SURF96 R C T  0 34.000    4.1568    1.0000    
SURF96 R U T  0 32.000    3.1906    1.0000    
SURF96 R C T  0 32.000    4.0868    1.0000    
SURF96 R U T  0 30.000    3.1341    1.0000    
SURF96 R C T  0 30.000    4.0133    1.0000    
SURF96 R U T  0 29.000    3.1092    1.0000    
SURF96 R C T  0 29.000    3.9755    1.0000    
SURF96 R U T  0 28.000    3.0881    1.0000    
SURF96 R C T  0 28.000    3.9370    1.0000    
SURF96 R U T  0 27.000    3.0724    1.0000    
SURF96 R C T  0 27.000    3.8983    1.0000    
SURF96 R U T  0 26.000    3.0647    1.0000    
SURF96 R C T  0 26.000    3.8596    1.0000    
